packages/datatrak-web/src/features/Survey/Components/SurveyQuestionGroup.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import styled from 'styled-components';
+import { Typography } from '@material-ui/core';
+import { SurveyQuestion } from './SurveyQuestion';
+import { SurveyScreenComponent } from '../../../types';
+
+const QuestionWrapper = styled.div<{
+  $isInstruction: boolean;
+}>`
+  display: flex;
+  &:not(:last-child) {
+    margin-bottom: ${({ $isInstruction }) => ($isInstruction ? '1rem' : '2rem')};
+  }
+`;
+
+const QuestionNumber = styled(Typography)`
+  width: 3.5rem;
+  text-transform: lowercase;
+`;
+
+/**
+ * This is the component that renders questions in a survey.
+ */
+export const SurveyQuestionGroup = ({ questions }: { questions: SurveyScreenComponent[] }) => {
+  return (
+    <>
```

Comment:
```suggestion

```
Unnecessary fragment

Reply:
on second thoughts this might be necessary to keep typescript happy, in which case no worries

Reply:
Yeah I think it has to be there for TS